docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -138,7 +138,7 @@ You can create new queries to fetch annotations from the built-in annotation que
 
 Grafana v8.1 and later versions also support typeahead of existing tags, provide at least one tag.
 
-For example, create an annotation query name `outages` and specify a tag `outage`. This query will show all annotations (from any dashboard or via API) with the `outage` tag. If multiple tags are defined in an annotation query, then Grafana will only show annotations matching all the tags. To modify the behavior, enable `Match any`, and Grafana will show annotations that contain any one of the tags you provided.
+For example, create an annotation query name `outages` and specify a tag `outage`. This query will show all annotations (from any dashboard or via API) with the `outage` tag. If multiple tags are defined in an annotation query, then Grafana will only show annotations matching all the tags. To modify the behavior, enable `Match any`, and Grafana will show annotations that contain any one of the tags you provided. You can also adjust the max number of  annotions shown by  modifying the `Max limit` from its default 100. 
 
 {{< figure src="/media/docs/grafana/dashboards/screenshot-annotations-typeahead-support-10.0.png" max-width="600px" >}}
```

Comment:
I've moved this content to after the image since the image illustrates the example I want that connection to be clearer. I also broke out the max number part that you added because it's a new topic.
```suggestion
{{< figure src="/media/docs/grafana/dashboards/screenshot-annotations-typeahead-support-10.0.png" max-width="600px" >}}

If you define multiple tags in an annotation query, then Grafana will only show annotations matching all the tags. To modify this behavior, enable the **Match any** toggle; then Grafana will show annotations that contain any of the tags you provided. 

You can also adjust the maximum number of annotations shown by modifying the **Max limit** from its default of `100`. 
```